front-end/src/utils/NotifyHeader.js

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import styled from "styled-components";
+import { StyledHeader } from "./styles";
+import { ColoredButton } from "../AdminBar/adminStyles";
+import { NotifyContainer, FormContainer, WaitInput } from "./styles";
+import bell from "./bell.png";
```

Comment:
I suggest using an SVG for this instead :)

Reply:
I drew my own svg! :D

Reply:
Woho, nice! What's cool about SVG is that it's just XML, so here's a "cleaned" version of the same icon :)

```xml
<svg width="400" height="400" xmlns="http://www.w3.org/2000/svg">
  <ellipse ry="35" rx="40.5" cy="349" cx="200" fill="#FABA6F"/>
  <path fill="#f9eae1" d="M84.5 167h239v187h-239zM173.383 39.465c0-12.979 11.864-23.5 26.5-23.5 7.028 0 13.768 2.476 18.738 6.883 4.97 4.407 7.762 10.384 7.762 16.617 0 12.979-11.865 23.5-26.5 23.5-14.636 0-26.5-10.521-26.5-23.5zm13.25 0c0 6.49 5.932 11.75 13.25 11.75 7.317 0 13.25-5.26 13.25-11.75s-5.933-11.75-13.25-11.75c-7.318 0-13.25 5.26-13.25 11.75z"/>
  <ellipse ry="130" rx="119" cy="176" cx="203.5" fill="#f9eae1"/>
  <ellipse ry="1" rx="7" cy="254" cx="310.5" fill="#f9eae1"/>
</svg>
```

Reply:
how did you do that? my cleanup stopped being a picture :(

Reply:
Oh noes! Not at my sharpest state right now, but the tool you want to use is called [`svgo`](https://github.com/svg/svgo). There's a nice webapp for it as well, called ["SVGOMG" (svgo's Missing GUI)](https://jakearchibald.github.io/svgomg/)

Reply:
here i though you did it by hand ;) cool tool